src/containers/ToDo.js

```diff
@@ -0,0 +1,105 @@
+import React, { Component } from "react";
+import AddToList from "../components/buttons/AddToList";
+import ResetList from "../components/buttons/ResetList";
+import AddListForm from "./AddListForm";
+import ToDoList from "../components/ToDoList/ToDoList";
+
+class ToDo extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      toDoList: [],
+      formStatus: false,
+      taskCounter: 0
+    };
+  }
+
+  componentDidMount() {
+    this.getDataFromLocal();
+  }
+
+  componentDidUpdate() {
+    this.setDataToLocal();
+  }
+
+  getDataFromLocal = () => {
+    if ("data" in localStorage) {
+      const data = JSON.parse(localStorage.getItem("data"));
+
+      this.setState({ toDoList: data.toDoList, taskCounter: data.taskCounter });
+    }
+  };
+
+  setDataToLocal = () => {
+    localStorage.setItem("data", JSON.stringify(this.state));
+  };
+
+  handleFormSubmit = (e, formData) => {
+    e.preventDefault();
+    const newTask = {
+      id: this.state.taskCounter + 1,
+      task: formData.task,
+      daysAssigned: formData.daysAssigned,
+      issueTimeStamp: new Date().getTime(),
+      isDone: false
+    };
+    const newTaskList = this.state.toDoList;
+    newTaskList.push(newTask);
+    this.setState(prevState => ({
+      toDoList: newTaskList,
+      formStatus: false,
+      taskCounter: prevState.taskCounter + 1
+    }));
```

Comment:
break down this method with multiple task into different small method that handles a single task.